app/imports/ui/pages/SignSmartContract.jsx

```diff
@@ -0,0 +1,134 @@
+import SignSmartContractItem from '../components/SignSmartContractItem';
+
+const contractSchemaSignature = new SimpleSchema({
+  signature: String,
+});
+
+const contractSchemaTenantStance = new SimpleSchema({
+  tenantStance: {
+    type: String,
+    allowedValues: ['Unsigned', 'Agreement'],
+    defaultValue: 'Unsigned',
+  },
+});
+
+const bridgeSignature = new SimpleSchema2Bridge(contractSchemaSignature);
+const bridgeTenantStance = new SimpleSchema2Bridge(contractSchemaTenantStance);
+
+const isTenant = (contract, username) => contract.tenantEmail === username;
+const isHomeowner = (contract, username) => contract.homeownerEmail === username;
+
+const missingSignature = (contract, username) => (isTenant(contract, username) && contract.tenantSignature === '') || (
+  isHomeowner(contract, username) && contract.homeownerSignature === '');
+
+/** Renders a table containing all of the Stuff documents. Use <SmartContractItem> to render each row. */
+class SignSmartContract extends React.Component {
+  // On successful submit, insert the data.
+
+  submit(data) {
+    const { tenantEmail, tenantStance, _id } = data;
+    const username = this.props.user.username;
+
+    if (username === tenantEmail) {
+      SmartContracts.collection.update(_id, { $set: { tenantStance } }, (error) => (error ?
+        swal('Error', error.message, 'error') :
+        swal('Success', 'Information updated successfully', 'success')));
+    } else {
+      swal('Error', 'Only tenant can fill out this field', 'error');
+    }
+  }
+
+  submitSignature(data) {
+    const { signature, _id, homeownerEmail, homeownerName, tenantEmail, tenantName, tenantStance } = data;
+    const username = this.props.user.username;
+
+    if (username === homeownerEmail && signature === homeownerName) {
+      SmartContracts.collection.update(_id, { $set: { homeownerSignature: signature } }, (error) => (error ?
+        swal('Error', error.message, 'error') :
+        swal('Success', 'Smart contract successfully signed by homeowner', 'success')));
+    } else if (username === tenantEmail && signature === tenantName) {
+      if (tenantStance === 'Agreement') {
+        SmartContracts.collection.update(_id, { $set: { tenantSignature: signature } }, (error) => (error ?
+          swal('Error', error.message, 'error') :
+          swal('Success', 'Smart contract successfully signed by tenant', 'success')));
+      } else {
+        swal('Error', 'Tenet stance must be agreement before signing', 'error');
+      }
+    } else {
+      swal('Error', 'Signature must be full name', 'error');
+    }
+  }
+
+  // If the subscription(s) have been received, render the page, otherwise show a loading icon.
+  render() {
+    return (this.props.ready) ? this.renderPage() : <Loader active>Getting data</Loader>;
+  }
+
+  // Render the page once subscriptions have been received.
+  renderPage() {
+    return (
+      <Grid container centered>
```

Comment:
Containers would work as well. Either way , maybe add 5% padding or margin on bottom and maybe top of your content.